src/styles/card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img-wrapper {
+    width: 160px;
+    height: 134px;
+    margin: 32px 20px 24px;
+  }
+
+  &__img {
+    width: 100%;
+  }
+
+  &__content {
+    margin-bottom: 16px;
+    width: 166px;
+
+  }
+
+  &__title {
+    color: #060b35;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    color: #616070;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin: 0;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 7px;
+  }
+
+  &__reviews-number {
+    color: #060b35;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__price-wrapper {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 0;
+  }
+
+  &__price {
+    color: #616070;
```

Comment:
color '#616070' is duplicated, please move all duplicated colors and size to the variables